app/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/views/templates/injected/TextInputHelper.scala.html

```diff
@@ -0,0 +1,47 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+        govukInput: GovukInput
+)
+
+@(      field: Field,
+        label: String,
+        isLabelHidden: Boolean = false,
+        isPageHeading: Boolean = true,
+        hint: Option[String] = None,
+        classes: Option[String] = None
+)(implicit messages: Messages)
+
+
+@govukInput(
+    Input(
+        label = Label(
+            isPageHeading = isPageHeading,
+            classes = if(isPageHeading) "govuk-label--l"
+            else if(isLabelHidden)"govuk-visually-hidden"
+            else "govuk-!-font-weight-bold",
```

Comment:
Can you tab these two lines in once more to indicate they are part of the if statement, could be confused as being part of Label itself